pyrepl/curses.py

```diff
@@ -19,5 +19,8 @@
 # CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 # CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-
-from ._minimal_curses import setupterm, tigetstr, tparm, error
+# First try the built-in module
```

Comment:
This should also mention that the builtin is only expected to be there with PyPy probably.